packages/web-client/app/components/card-pay/create-merchant-workflow/merchant-customization/index.hbs

```diff
@@ -1,27 +1,28 @@
 <ActionCardContainer
   @header="Merchant Account"
   @isComplete={{@isComplete}}
-  data-test-layout-customization-is-complete={{@isComplete}}
+  data-test-merchant-customization-is-complete={{@isComplete}}
 >
   <ActionCardContainer::Section @title="Choose a name and ID for the merchant">
     <CardPay::LabeledValue
       class="merchant-customization__name"
       @label={{if @isComplete "Merchant Name" "Preview"}}
+      data-test-merchant-customization-merchant-preview
```

Comment:
I think this can be addressed separately but we should figure out what will and won’t work here:

![image](https://user-images.githubusercontent.com/43280/130256469-0584bb42-53be-457a-aa53-4fc8774384f5.png)

![image](https://user-images.githubusercontent.com/43280/130256796-4227adaf-dfff-41d6-9cb5-abebabf5cdb9.png)

![image](https://user-images.githubusercontent.com/43280/130256864-e88fe0be-24db-41bd-94a4-96dbc33fca19.png)
[DIĜIR](https://en.wikipedia.org/wiki/Dingir)

![image](https://user-images.githubusercontent.com/43280/130256971-5ef08df7-803e-494b-aa28-9e7432310b43.png)

![image](https://user-images.githubusercontent.com/43280/130257601-ba56f03e-8202-40cc-be93-2073eb84819d.png)
[BELL](http://www.unicode-symbol.com/u/0007.html) is considered valid but appears blank

Reply:
hmm we should update the `first-char` helper to account for these. Currently it only accepts string type.

Reply:
@backspace What do you recommend for checking for emojis etc inside a string? Regex?

Reply:
I found this:

https://medium.com/@giltayar/iterating-over-emoji-characters-the-es6-way-f06e4589516

which suggests using for-of loops or `Array.from(str)`, these implementations account for unicode. I can make this change @burieberry

Reply:
ah I didn’t realise this was about `first-char`! I thought it was just the font. Nice find re for/of 🎉

Reply:
> I found this:
> 
> https://medium.com/@giltayar/iterating-over-emoji-characters-the-es6-way-f06e4589516
> 
> which suggests using for-of loops or `Array.from(str)`, these implementations account for unicode. I can make this change @burieberry

Sure, sounds good!

Reply:
https://github.com/cardstack/cardstack/pull/1980/commits/389374446a2b2f60449e8c886cccf2daf83666a0 solves the emoji/multi-code-unit issue, but characters like BELL that don't have a visual representation are a bit trickier. Will create a follow up PR for some input-filtering and maybe whitespace-collapsing.